calico_containers/tests/st/utils/defaults.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2015 Metaswitch Networks
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# IPv4 Pool Adresses
+DEFAULT_IPV4_POOL = "192.168.0.0/16"
+IPV4_POOL_ADDR_1 =  "192.168.1.1"
+IPV4_POOL_ADDR_2 =  "192.168.1.2"
+
+# IPv6 Pool Addresses
+DEFAULT_IPV6_POOL = "fd80:24e2:f998:72d6::/64"
+IPV6_POOL_ADDR_1 = "fd80:24e2:f998:72d6::11"
+IPV6_POOL_ADDR_2 = "fd80:24e2:f998:72d6::22"
```

Comment:
Whoops, ADDR_1 and ADDR_2 aren't used here, I copied them from another test I wrote, meant to get rid of them.

There isn't a `defaults` or `constants` file for the calico pools.  These pools are general also, not just libnetwork.  

I decided to put these default pool addresses in an ST file in case we decide to use more ST constants that are unrelated to the rest of the repo (as the ADDR_1 and ADDR_2 values would be).  I would also be on board with saving the IP as a global wherever the default pool is defined in calico-docker, though I think have a defaults file for the STs is also useful.